src/transaction.ts

```diff
@@ -0,0 +1,234 @@
+import { Psbt, networks } from 'liquidjs-lib';
+import { estimateTxSize, UtxoInterface } from './wallet';
+
+export interface OutputInterface {
```

Comment:
@tiero how much abstraction do we want here? We could use the `address` here instead of the `script`... otherwise the user would need to convert by himself the receiving addresses into scripts (it would end up using liquidjs-lib for that).